espnet/nets/pytorch_backend/conformer/encoder.py

```diff
@@ -93,6 +93,8 @@ def __init__(
         padding_idx=-1,
         stochastic_depth_rate=0.0,
         intermediate_layers=None,
+        ctc_softmax=None,
+        odim=None,
```

Comment:
I prefer to use `conditioning_layer_dim` instead of `odim` here. Although the output dimension of CTC is vocabulary size (`odim`), using another name like `conditioning_layer_dim` may reduce ambiguity. It is okay to keep `odim`.

Reply:
I will change it to `conditioning_layer_dim`.